### ISSUE TITLE
Delete all cookies across all domains

### ISSUE THREAD

**Issue by dougleenhouts** [a85/POSTMan-Chrome-Extension#12053]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

I have never been able to programmatically delete all cookies for a domain via a script. I've whitelisted all possible variations, such as:

platform.domain.com
*.platform.domain.com
*.domain.com
domain.com
https://platform.domain.com

But I've never seen this well-documented script actually clear the cookie jar, delete a single cookie, or even unset one. 

const jar = pm.cookies.jar();

```
jar.clear(pm.request.url, function (error) {
  // error - <Error>
});
```

I would like a script that does the equivalent of clicking the green-circled "X". Would it be possible for jar.clear() to accept a domain rather than a URL (assuming it even works as designed)?

![Screenshot 2023-05-25 at 5 31 50 PM](https://github.com/postmanlabs/postman-app-support/assets/107500230/3265a329-5e8b-48d6-bfe4-02a1a7c23591)

### Describe the solution you'd like

I'd like a script or a function that can completely nuke every single cookie for every single domain, that results in the same state as a fresh installation of the postman desktop client.

**Comment by giridharvc7** [comment]
Have you checked this `Clear All Cookies` in the Cookie editor

![image](https://github.com/postmanlabs/postman-app-support/assets/1040583/59381b12-409d-4e7c-bc71-9d54dc4445db)

**Comment by dougleenhouts** [comment]
Yes, I want that functionality in either a pre-request script or a test script, not only as a UI option.

**Comment by adarshtaravind1** [comment]
![Screenshot 2024-03-25 at 5 40 03 PM](https://github.com/postmanlabs/postman-app-support/assets/159620724/c333b7c4-5104-43a7-9f16-bdeef82a056a)
Use this option under settings. it will get automatically cleared before each hit

**Comment by WarrDaddy** [comment]
export your collection into json file and all this param to enable the "Disable cookiejar" settings then import it back. I used chatgpt to herlp me out
